.gitignore

```diff
@@ -1,7 +1,7 @@
 .php_cs.cache
 composer.phar
 composer.lock
+phpunit.xml
```

Comment:
@dunglas is that really an issue? It's not IDE or OS specific and does not cost much to be added. It's pretty much like the `parameters.yml.dist` and `parameters.yml` for Symfony.